code/config.py

```diff
@@ -20,6 +20,8 @@ def debug(message):
     # if the os is windows, we logging.info(message), if
     if os.name == "nt":
         logging.info(message)
+    else:
```

Comment:
Why? It's supposed to debug only when testing. I should add a .env debug variable instead.

Reply:
Yup, definitely
And as for the reason, it's to print errors on stdin while in docker container